edgelet/iotedged/src/lib.rs

```diff
@@ -2074,11 +2076,14 @@ where
     );
 
     env.insert(WORKLOAD_URI_KEY.to_string(), workload_uri);
-    #[cfg(feature = "runtime-docker")]
-    env.insert(
-        WORKLOAD_LISTEN_MNT_URI.to_string(),
-        Listen::workload_mnt_uri(&home_dir),
-    );
+    if env::var(IOTEDGE_WORKLOAD_SOCKETS_ENABLE).unwrap_or_else(|_| "False".to_string()) == "True" {
+        #[cfg(feature = "runtime-docker")]
```

Comment:
This `#[cfg]` should be on the if-block itself rather than just the statement inside it.

Reply:
Fixed